site/content/en/docs/Integration Patterns/reusing-gameservers.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Reusing Allocated GameServers for more than one game session"
+linkTitle: "Reusing Gameservers"
+date: 2021-09-01
+weight: 50
+description: >
+  After a `GameServer` has completed a player session, return it back to the pool of Ready `GameServers` for reuse. 
+---
+
+Having a `GameServer` terminate after a single player session is better for packing and optimisation of 
+infrastructure usage, as well as safety to ensure the process returns to an absolute zero state.
+
+However, depending on the `GameServer` startup time, or other factors there may be reasons you wish to reuse a 
+`GameServer` for _n_ number of sessions before finally shutting it down.
+
+The "magic trick" to this is knowing that the `GameServer` process can call 
+[`SDK.Ready()`]({{% ref "/docs/Guides/Client SDKs/_index.md#ready" %}}) to return to a `Ready` 
+state after the `GameServer` has been allocated. 
+
+It is then up to the game developer to ensure that the game server process returns to a zero state once a game 
+session has been completed. 
+
+![Reserved Lifecycle Sequence Diagram](../../../diagrams/reusing-gameservers.puml.png)
+
+## Next Steps
+
+- Have a look at all commands the [Client SDK]({{< ref "/docs/Guides/Client SDKs/_index.md" >}}) provides.
+- Check out the answer to the FAQ's question:
```

Comment:
This FAQ seems less useful and descriptive than the new integration pattern page being added in this PR. I think it might make more sense to remove this link and have the FAQ point to this page instead.

Reply:
That's an excellent suggestion. Let me do that 👍🏻